src/ada_idna.cpp

```diff
@@ -2632,7 +2632,9 @@ uint32_t find_range_index(uint32_t key) {
 }
 
 bool ascii_has_upper_case(char* input, size_t length) {
-  auto broadcast = [](uint8_t v) -> uint64_t { return 0x101010101010101 * v; };
+  auto broadcast = [](uint8_t v) -> uint64_t {
+    return 0x101010101010101ull * v;
```

Comment:
Can you also update `ada/idna` repository with the relevant changes as well?

Reply:
yes, i can do that once we've finished this one